build.sh

```diff
@@ -1,11 +1,14 @@
-#!/bin/bash
+#!/bin/bash -eu
 
 PATH=/opt/arm/gcc-linaro-7.5.0-2019.12-i686_arm-linux-gnueabihf/bin:$PATH
 export INSTALL_MOD_PATH=$(pwd)/modules
 export ARCH=arm
 export CROSS_COMPILE=arm-linux-gnueabihf-
 
-make menuconfig
+#make savedefconfig
+#make menuconfig
```

Comment:
Can you remove commented out lines?

Reply:
No. All of the commented lines in this file need to remain in the file. This is a template file that should only be edited locally and never re-committed (except when I do so by mistake). The purpose of this file is to give you a simple template for building.